src/features/plans/components/CategoryCard.tsx

```
@@ -0,0 +1,110 @@
+  const { label, Icon } = categoryMeta[type]
+  return (
+    <div
+      className={cn(
+        'rounded-xl border border-border/50 bg-card transition-all',
+        isExpanded && 'ring-1 ring-primary/20',
+        isDragging && 'opacity-50 scale-[0.98]',
+      )}
+    >
+      {/* 카테고리 헤더 */}
+      <div className="flex items-center gap-3 p-4 cursor-pointer" onClick={onToggle}>
+        <div className="flex h-8 w-8 items-center justify-center text-muted-foreground cursor-grab active:cursor-grabbing hover:text-foreground transition-colors">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)